index.html

```diff
@@ -687,10 +686,10 @@
               <code>zoom</code>
             </dt>
             <dd>
-              Zooms the content corresponding the accessible the action
-              is invoked on. Extra parameters: ratio of double. Values great than
-              1 zooms in the area, values lesser than zooms out it. 1 brings the area
-              to its normal size.
+              Zooms the content corresponding the accessible the action is
+              invoked on. Extra parameters: ratio of double. Values great than
+              1 zooms in the area, values lesser than zooms out it. 1 brings
```

Comment:
Maybe be a bit more explicity. 
values lesser than 1 than zooms out it. 

What happens when the number is less than 0? or greater than 1?